.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
There are some files added by accident here.

Reply:
Thank you for catching this!!